request_a_govuk_domain/request/migrations/0001_initial.py

```diff
@@ -66,7 +66,7 @@ class Migration(migrations.Migration):
                                 "Town, county, borough, metropolitan or district council",
                                 "Local Authority",
                             ),
-                            ("Fire Service", "Fire Service"),
+                            ("Fire service", "Fire Service"),
```

Comment:
I think we should make it clearer which of the pairs is human-readable and which is for code, as per https://docs.djangoproject.com/en/5.0/ref/models/fields/#choices
So here we should have something like `("FIRE", "Fire Service")` -- or the other way around, or `("JOINT", "Joint Committee")` below.